WebDriverAgent.xcodeproj/xcshareddata/xcschemes/WebDriverAgentRunner.xcscheme

```diff
@@ -75,6 +75,11 @@
             value = "$(COMMIT_HASH)"
             isEnabled = "YES">
          </EnvironmentVariable>
+         <EnvironmentVariable
```

Comment:
Why an environment variable and not a capability?

Reply:
because this setting is global. The broadcasting thread starts together with the WDA agent's web server

Reply:
Ah! Makes sense.